relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -0,0 +1,693 @@
+//! Logic for persisting items into `span.data` fields.
+//! These are then used for metrics extraction.
+
+use itertools::Itertools;
```

Comment:
_nit_: What is itertools used for? I wasn't able to spot it while scanning through the file and wonder if we can go without.

Reply:
It's used to `.join()` an iterator [into a string](https://github.com/getsentry/relay/pull/2304/files#diff-53f81fab03a0fb9fa97597b16f42b28ec40c153b47ee77f03b67580f89081302R550).